### ISSUE TITLE
Staging Review finding: va-radio-button imposter component

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#111618]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Unified Appointment Experience
**Product:** Appointments
**Feature:** Community Care Direct Scheduling

## Findings details
**VA.gov Experience Standard - issue:** [User encounters components and patterns that meet best practices.](https://design.va.gov/about/experience-standards/#user-encounters-components-and-patterns-that-meet-best-practices)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @rsmithadhoc (Accessibility)

### Description

On "Schedule an appointment with your provider > Choose a date and time", the radio button is not rendered by va-radio-button.

<details><summary>Link, screenshot or steps to recreate</summary>

(to be added imposter-4)

</details>

## Recommended action

Use the VADS component.

## References

- **Accessibility Defect Severity:** N/A
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** [Form - Radio button component](https://design.va.gov/components/form/radio-button)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by wullaski** [comment]
@outerpress we probably have time for this next week.

**Comment by sixercjl** [comment]
@outerpress , @wullaski and I took a look at this component and it's highly customized and won't be a clean cutover to the VA radio button component. We would need to override styles inside the VA web component which seems to contradict the point of using the new component. How do you think we should proceed?

https://github.com/department-of-veterans-affairs/vets-website/blob/bcdab154dc12b5adaa8597f03abf919b615f894e/src/applications/vaos/components/calendar/CalendarOptionsSlots.jsx#L138

**Comment by outerpress** [comment]
@wullaski @a6marinos @StevenStraily I have to apologize - I intended to push forward the issue with the same name, but for the radio component on the type of care page - https://github.com/department-of-veterans-affairs/va.gov-team/issues/111616

I'm sorry I sent you all down the wrong path. And I'm also thankful that you documented the details of why we won't be working on the calendar widget here. I'll reach out separately in the type of care ticket.
